src/app/app.module.ts

```diff
@@ -70,7 +71,10 @@ import { CalloutComponent } from './components/callout/callout.component';
     BrowserAnimationsModule,
     MdSliderModule,
     routing,
-    store
+    store,
+    StoreDevtoolsModule.instrumentOnlyWithExtension({
```

Comment:
I tried adding the store to the redux devtools to try to figure out what is going on in this application, but the devtools weren't recognizing anything. I'm not sure if this app is actually using a store - it looks like there was one built (perhaps retroactively?)